src/updateIndices.js

```diff
@@ -0,0 +1,9 @@
+const updateIndices = () => {
+  const data = JSON.parse(localStorage.getItem('data'));
+  data.forEach((todo, index) => {
+    todo.index = index + 1;
+  });
+  localStorage.setItem('data', JSON.stringify(data));
+};
```

Comment:
- [ ] You have done a great job on this milestone but there are still some issues you still need to address to complet the project,
- [ ] I would suggest you to add functionality to the clear `clear all completed` button as required on the project requirement by Removing all items marked as complete at once.
- [ ] see your Todo-List App
 

https://github.com/enocol/webpack/assets/92458236/b66d8a44-abc8-492f-b7b9-4d1b985c493c

- [ ] Note that the user can delete items one by one and also the user can mark all items and delete them all at once, please fix it I believe you can do it perfectly.

Reply:
@ademibowale I thought that was the requirement for the day 4 project. I have submitted the day 3 project. Please look at the project requirements for the day of the project and advise accordingly;.